lib/Dialect/Torch/Transforms/GlobalizeObjectGraph.cpp

```diff
@@ -97,6 +101,32 @@ class ObjectGraphInfo {
   }
 
 private:
+  LogicalResult collectUsedSlots() {
+    llvm::StringMap<llvm::StringMap<SlotOp>> moduleClassNameToSlots;
+    symbolTable.getOp()->walk([&](NnModuleOp moduleOp) {
+      llvm::StringMap<SlotOp> nameToSlot;
+      for (auto attrOp : moduleOp.getOps<SlotOp>())
+        nameToSlot[attrOp.name()] = attrOp;
+      moduleClassNameToSlots[moduleOp.getClassName()] = nameToSlot;
+    });
+
+    symbolTable.getOp()->walk([&](PrimGetAttrOp op) {
```

Comment:
Do you need to check PrimSetAttr too? I'm worried that if there are set but no get, the pass will continue and violate some invariant later. (could you add a test case with set and no get?)

Reply:
Good point!